### DISCUSSION TITLE
Classic queue storage format version 1 is deprecated in 3.13. Clarification on the word 'deprecated'

### DISCUSSION THREAD

**Discussion by SimonUnge** [rabbitmq/rabbitmq-server#10388] [Other] [answered]
Hi,

I have a minor question of clarification. In 3.13, cqv1 will be 'deprecated', which could mean "supported but no longer recommended, may eventually be phased out so please use something else".

Is that true for rabbitmq 3.13? I ask because I vaguely remember a harder line in the past, that cqv1 will be *removed* in 3.13.

Best regards

**Comment by lukebakken** [comment, marked as answer]
Hey @SimonUnge, does this blog post answer your question?

https://blog.rabbitmq.com/posts/2024/01/3.13-release/

> I have a minor question of clarification. In 3.13, cqv1 will be 'deprecated', which could mean "supported but no longer recommended, may eventually be phased out so please use something else". Is that true for rabbitmq 3.13?

Yes, 3.13 deprecates CQv1 but does not remove them.

**Reply by SimonUnge** [reply]
Oh, the blog post is clear, but my, perhaps faulty, memory of previous statements made me question the meaning of the word 'deprecated'!
